src/backlight/portfolio/create_portfolio.py

```diff
@@ -0,0 +1,23 @@
+# Define how to create Portfolio
```

Comment:
it is better to name module as noun. This file will contain codes for `Strategy for portfolio`. What is preferable name? 🤔

Reply:
Or, we can move it under `backlight/strategy`? 🤔

Reply:
maybe `build_portfolio` ? I think `backlight/strategy` contains strategies only for assets. It may be better to have everything related to portfolio in `backlight/portfolio`? 
If you have some specific convention for that, let me knot to make necessary changes.

Reply:
Could you name this file as `backlight/portfolio/strategy.py` now?